source/default_mode/Host.h

```diff
@@ -439,7 +439,17 @@ class Host: public Organism {
    * Purpose: To add a symbionts to a host's symbionts
    */
   int AddSymbiont(emp::Ptr<Organism> _in) {
-    if((int)syms.size() < my_config->SYM_LIMIT() && SymAllowedIn()){
+    bool allowed_in = SymAllowedIn();
+    if (my_config->OUSTING() && allowed_in && (int)syms.size() == my_config->SYM_LIMIT()) {
+      int new_sym_pos = (syms.size() > 1) ? random->GetInt(syms.size()) : syms.size()-1;

```

Comment:
I'm unfairly biased against ternary operators and have a hard time following them, can this be either just a normal if or have a comment explaining what it is doing?